src/tls/tls.h

```diff
@@ -17,27 +15,25 @@
 // Some MbedTLS errors are not matched by OpenSSL, so we set some ridiculous
 // negative number that will never match. This allows us to place the macro as
 // a switch-case and not duplicate cases but also never match.
-#  define TLS_READING -SSL_READING
-#  define TLS_WRITING -SSL_WRITING
-#  define TLS_ERR_WANT_READ -SSL_ERROR_WANT_READ
-#  define TLS_ERR_WANT_WRITE -SSL_ERROR_WANT_WRITE
-#  define TLS_ERR_CONN_CLOSE_NOTIFY -SSL_ERROR_ZERO_RETURN
-#  define TLS_ERR_NEED_CERT -SSL_ERROR_WANT_X509_LOOKUP
+#define TLS_READING -SSL_READING
+#define TLS_WRITING -SSL_WRITING
+#define TLS_ERR_WANT_READ -SSL_ERROR_WANT_READ
+#define TLS_ERR_WANT_WRITE -SSL_ERROR_WANT_WRITE
+#define TLS_ERR_CONN_CLOSE_NOTIFY -SSL_ERROR_ZERO_RETURN
+#define TLS_ERR_NEED_CERT -SSL_ERROR_WANT_X509_LOOKUP
 // No counterpart in OpenSSL
-#  define TLS_ERR_CONN_RESET INT_MIN
-#  define TLS_ERR_PEER_VERIFY INT_MIN + 1
-#  define TLS_ERR_X509_VERIFY INT_MIN + 2
+#define TLS_ERR_CONN_RESET INT_MIN
```

Comment:
We don't need to treat those errors anymore, so they could just be removed from the switch/cases.

Reply:
They are still used in a number of places. Are you sure we can remove all those switch/cases? I'm inclined to leave this to you :-)

Reply:
Awesome, thanks!

Reply:
#3418